src/helper/error.ts

```diff
@@ -0,0 +1,7 @@
+export const ERROR = {
+  ACCOUNT_NOT_SOURCE:
+    "Transfer contains authorization entry for a different account",
+  FOREIGN_CONTRACT_AUTH:
```

Comment:
did we mean to use this error in this pull request or are you planning to use it in a follow-up change?

Reply:
Actually I'll remove that one, @sisuresh pointed how that check was already covered.

Reply:
removed in 3272c3c01df00ef3db048298a44055acca7e0b97